app/src/main/java/com/gathering/android/common/Composables.kt

```diff
@@ -231,5 +243,65 @@ fun ProgressBar(
     }
 }
 
+@Composable
+@Preview(showBackground = true)
+fun ShowProfilePicturePreviewWithString() {
+    ShowImage(imageUri = "")
+}
+
+@Composable
+fun ShowImage(imageUri: String?) {
+    Card(
+        modifier = Modifier
+            .padding(25.dp)
+            .clip(CircleShape)
+            .border(2.dp, Color.Gray, CircleShape),
+        colors = CardDefaults.cardColors(
+            containerColor = Color.White
+        )
+    ) {
+        val painter = if (imageUri.isNullOrEmpty()) {
+            painterResource(id = R.drawable.ic_person)
+        } else {
+            rememberAsyncImagePainter(model = "https://moyar.dev:8080/photo/${imageUri}")
+        }
+        Image(
+            contentScale = ContentScale.Crop,
+            painter = painter,
+            contentDescription = "Profile picture",
+            modifier = Modifier
+                .size(200.dp)
+                .background(Color.White, CircleShape)
+        )
+    }
+}
+
+@Composable
+fun ShowImage(imageUri: Bitmap?) {
```

Comment:
naming the bitmap imageUri makes things very confusing here. is should be called bmp or something

Reply:
done